packages/@aws-cdk/aws-cloudfront/lib/web_distribution.ts

```diff
@@ -651,14 +705,50 @@ export class CloudFrontWebDistribution extends cdk.Construct implements IDistrib
 
     distributionConfig = { ...distributionConfig, cacheBehaviors: otherBehaviors };
 
-    if (props.aliasConfiguration) {
-      const minimumProtocolVersion = props.aliasConfiguration.securityPolicy;
-      const sslSupportMethod = props.aliasConfiguration.sslMethod || SSLMethod.SNI;
-      const acmCertificateArn = props.aliasConfiguration.acmCertRef;
+    if (props.aliasConfig || props.aliasConfiguration) {
+      // TODO:v2 remove props.aliasConfiguration usage, cleanup
+      const {
+        names: aliases,
+        securityPolicy: minimumProtocolVersion,
+        sslMethod: sslSupportMethod = SSLMethod.SNI
+      } = props.aliasConfig || props.aliasConfiguration!;
+
+      if (props.aliasConfig) {
+        const {acmCert} = props.aliasConfig;
+
+        const isDnsCert = (cert: certificatemanager.ICertificate): cert is certificatemanager.DnsValidatedCertificate =>
```

Comment:
I would rather have some static helper functions in the ACM module to get the region from an `ICertificate`. One module grubbing around in the internals of another one without a well-defined public interface to do so seems like a bad idea.

The simplest would be:

```ts
class CertificateUtils {
  public static certificateRegion(cert: ICertificate): string | undefined {
    // Put all that logic here
  }
}
```